styles/typography.css

```diff
@@ -16,3 +15,90 @@
   --font-family-serif: "Source Serif 4", serif;
   --bs-font-sans-serif: "Source Sans 3";
 }
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+.h1,
+.h2,
+.h3,
+.h4,
+.h5,
+.h6 {
+  font-weight: 600;
+}
+
+h6,
+.h6 {
+  text-transform: uppercase;
+}
+
+dt {
+  font-weight: 600;
+}
+
+dd {
+  font-weight: 400;
+}
+
+p {
+  font-size: 1rem;
+  font-weight: 400;
+}
+
+/* xs */
+@media (width < 576px) {
+  p {
+    font-size: 0.9rem;
```

Comment:
Isn't this a challenge for accessibility? Some experts suggest no smaller than 1rem for body text. https://accessibe.com/blog/knowledgebase/ada-compliant-fonts